rpm/plan.sh

```diff
@@ -0,0 +1,23 @@
+pkg_name=rpm
+pkg_origin=core
+pkg_version=4.12.0.1
+pkg_license=("GPL-2.0" "LGPL-2.0")
+pkg_description="RPM Package Manager"
+pkg_upstream_url=http://www.rpm.org/
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_upstream_url=http://rpm.org
+pkg_source=http://www.rpm.org/releases/${pkg_name}-4.12.x/${pkg_name}-${pkg_version}.tar.bz2
+pkg_shasum=77ddd228fc332193c874aa0b424f41db1ff8b7edbb6a338703ef747851f50229
+pkg_deps=(core/glibc core/zlib rarenerd/nspr rarenerd/nss core/file rarenerd/popt core/libarchive core/db)
+pkg_build_deps=(core/gcc core/make)
+pkg_bin_dirs=(bin)
+pkg_include_dirs=(include)
+pkg_lib_dirs=(lib)
+
+do_build() {
+  ./configure --prefix="${pkg_prefix}" \
+              --with-external-db       \
+              --without-lua
```

Comment:
Adding Lua to RPM allows someone writing a RPM spec file to use Lua scripting in the `%pre` and `%post` sections. As far as I can see it's not an often used feature, but I do think we should support this.

However at the moment I'm being blocked by a transient dependency issue. Package `core/sqlite/3130000/20160727232626` depends on `core/readline/6.3.8/20160727225040` and `core/lua/5.3.3/20160729235733` depends on `core/readline/6.3.8/20160729192706`. As discussed on Slack I can't get this working until one or both of these are rebuild.

So we can either wait and I'll update this PR once that is resolved, or I'll submit a PR later to add Lua.